paperkite-rubocop.gemspec

```diff
@@ -17,13 +17,14 @@ Gem::Specification.new do |spec|
     "config/rspec.yml"
   ]
 
-  spec.add_dependency "rubocop",             "~> 1.14.0"
-  spec.add_dependency "rubocop-performance", "~> 1.11.2"
-  spec.add_dependency "rubocop-rails",       "~> 2.10.1"
+  spec.add_dependency "rubocop",             "~> 1.43.0"
+  spec.add_dependency "rubocop-performance", "~> 1.15.2"
+  spec.add_dependency "rubocop-rails",       "~> 2.17.4"
   spec.add_dependency "rubocop-rspec",       "~> 2.3.0"
 
-  spec.add_development_dependency "rubocop",             "~> 1.14.0"
-  spec.add_development_dependency "rubocop-performance", "~> 1.11.2"
-  spec.add_development_dependency "rubocop-rails",       "~> 2.10.1"
-  spec.add_development_dependency "rubocop-rspec",       "~> 2.3.0"
+  spec.add_development_dependency "rubocop",             "~> 1.43.0"
+  spec.add_development_dependency "rubocop-performance", "~> 1.15.2"
+  spec.add_development_dependency "rubocop-rails",       "~> 2.17.4"
+  spec.add_development_dependency "rubocop-rspec",       "~> 2.17.0"
+  spec.metadata['rubygems_mfa_required'] = 'true'
```

Comment:
If you are needing MFA for RubyGems please make sure to put it into the 1Password account. It doesn't currently have MFA.

https://start.1password.com/open/i?a=UT24R7LQYBCCFDK33QA65KNIXA&v=7my2ezhpe5gknb4ww3tlgeog2a&i=c43dindlavgubfztrnrakiwvj4&h=paperkite.1password.com

Reply:
@invke I did change it "false" for now as new versions required to have MFA: https://guides.rubygems.org/mfa-requirement-opt-in/

Reply:
Shot, thanks @deep12650  🤙

Reply:
Just out of curious as I don't think I've ever actually updated version. @deep12650 , did you read the document and added it manually? Or it's automatically added when you updated the version?